### PR TITLE
#10 POST comments tested for happy and sad path

### PR DESCRIPTION
**Pull request by Habiba220**
- posts new comment and returns it
- tested for: 404, 400 for article id
- tested for 404 username not found
- tested for 400 username or body invalid type
- tested for 400 missing content for username or body

### CODE REVIEW

File: model/users.model.js [hunk 1]
```diff
@@ -1,9 +1,22 @@
+const { rows } = require("pg/lib/defaults");
```
Comment (Rosem10) [inline model/users.model.js]: Oooh don't like this! (looks like a pesky auto import to me - please remove!)

File: __tests__/app.test.js [hunk 2]
```diff
@@ -276,5 +276,87 @@ describe('GET /api/', () => {
...
+            const body = {username: "icellusedkars", body: 'nice article'}
+
+            return request(app)
+            .post('/api/articles/3/comments')
+            .send(body)
+            .expect(200)
+            .then(({ body : {new_comment} }) => {
+                expect(new_comment).toEqual(
+                    expect.objectContaining({
+                        comment_id: expect.any(Number),
```
Comment (Rosem10) [inline __tests__/app.test.js]: I think we can explicitly say that we expect this to be 3.

**Review by Rosem10 — COMMENTED** [review]
Good job! Couple of tiny tweaks but you can include them in the PR for the next ticket if you like 👍

**Review by Rosem10 — APPROVED** [review]

File: __tests__/app.test.js (hunk 2 above)
Comment (Habiba220) [reply __tests__/app.test.js]: Will do thanks Rose!

**Pull request merged**